### DISCUSSION TITLE
[QT6] Cannot build on Linux(at least on openSUSE Tumbleweed) : qpa/qplatformnativeinterface.h and qpa/qplatformwindow.h not found but exists in the system.

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#3856] [Q&A] [answered]
### What happened?

86Box can't be build anymore when using Qt6 on Linux because it can't find qpa/qplatformwindow.h and qpa/qplatformnativeinterface.h, these files exists ony my system.

These files are stored in : /usr/include/qt6/QtGui/6.6.0/QtGui/qpa/*.h

### Configuration file

```ini
No config file as it's a build issue not software issue.
```

### Operating system

openSUSE Tumbleweed [Linux 6.6.3]

### CPU

Intel Core i5-12450H

### 86Box version

master branch[5 December 2023]

### Build architecture

Linux - x64 (64-bit)

### Build type

- [X] New recompiler
- [ ] Debug build

### Download source

I built 86Box myself (please tell us more about your build configuration)

### Additional context

I tried to build 86Box on my linux system with these commands :

cmake -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DCMAKE_LINKER=/usr/bin/ld.lld -GNinja

ninja -C build -j12

**Comment by OBattler** [comment]
Make sure the files are in the correct path (check your include folder).

**Comment by OBattler** [comment, marked as answer]
Please follow the build guide.

**Reply by cold-brewed** [reply]
There are also several unnecessary cmake flags being specified which should be automatically detected.

If you want to build with QT 6 you will also need `-D USE_QT6=ON`.

**Comment by ghost** [comment]
Note : I forgot to add the -DUSE_QT6=ON flag on my bug report but it was there in the command I executed.

And the files the compiler can't find are in the /usr/include/qt6/QtGui/qpa/*.h
